code/modules/vehicles/armored/armored.dm

```diff
@@ -0,0 +1,745 @@
+ * The layers set this high because multitile sprites have issues with lighting otherwise
+ * Also % damage overlays are calculated on damage so remember to add those
+ *
+ * MULTITILE:
+ * Since we cant make non 1x1 vehicles move normally we instead make a hitbox that is forcemoved onto the location of the tank,
+ * creating a dense area while allowing the tank to move. The Hitbox also acts as a relay for projectiles, all other interactions are handled
+ * by Adjacent().
+ * Doorpoints exist for both multitile and 1x1 and determine the location where the humans are allowed to go in
+ * Both doorpoints and multitile are tracked in a list meaning that you can make non-rectangular vehicles by layering hitboxes
+ * And multiple enterances through which humans can get in
+ * Crushing is calculated on relaymove, where we select a main "centerturf" ahead that we can then calculate the rest of the tiles from,
+ * Although you can alter the exact method to suit your needs
+ */
+
+ /* TIVIS GUIDE TO MAKING VEHICLES, OR THE DUMMIES GUIDE TO TANK MAKING
+  *
+  * First get your sprites in
+  * You must manually set the offsets for your tank subtypes so that they move correctly, VV them in game to find the perfect values for it
+  * add default weapons and decide whether it should move diagonally or not
+  * add a doorpoint for where you want people to get in from using get_door_location()
+  *
+  * MULTITILE EXTRAS:
+  * Set the hitbox_type to your hitbox, /hitbox is 3x3 and /hitbox/medium is 2x2 so make custom ones as required
+  * you can also layer multiple hitboxes to make ... special shapes
+  * REMEMBER, USE ONLY BOUNDS DIVISIBLE BY 32 OR GLIDING WILL BREAK
+  * Set doorpoints for where you want access from, set this in the get_door_location() proc
+  * Set relaymove like this:
+  * First do checkmove, this checks whether the vehicle is ready to move
+  * Then calculate centerturf and the rest of the tiles that need to be rammed using get_steps() and turns()
+  * Then do do_move(), which handles ramming of the actual turfs
+  */
+
+
+/obj/vehicle/armored
+	name = "/improper TAV - Shortstreet MK4"	//PEAK PERFORMANCE MINITANK
+	desc = "An adorable chunk of metal with an alarming amount of firepower designed to crush, immolate, destroy and maim anything that Nanotrasen wants it to. This model contains advanced Bluespace technology which allows a TARDIS-like amount of room on the inside."
+	icon = 'icons/obj/tinytank.dmi'
+	icon_state = "tank"
+	pixel_x = -16
+	pixel_y = -8
+	layer = ABOVE_MOB_LAYER
+	anchored = FALSE
+	req_access = list(ACCESS_MARINE_TANK)
+	move_delay = 0.7 SECONDS
+	obj_integrity = 1000 //Friendly fire resistant
+	max_integrity = 1000
+	obj_flags = PREVENT_CONTENTS_EXPLOSION
+	anchored = TRUE //No bumping / pulling the tank
+	demolish_on_ram = TRUE
+	soft_armor = list("melee" = 50, "bullet" = 60, "laser" = 30, "energy" = 20, "bomb" = 0, "bio" = 20, "rad" = 0, "fire" = 50, "acid" = 50)
```

Comment:
Having a lower bio armor makes staying sentinal beneficial to xenos. I'd swap the bio and acid values.